chromium/popup.js

```diff
@@ -77,9 +77,14 @@ function createRuleLine(ruleset) {
   return line;
 }
 
-// Change the UX to reflect extension enabled/disabled
-function updateEnabledDisabledButtonState() {
+// Change the UI to reflect extension enabled/disabled
+function updateEnabledDisabledUI() {
   document.getElementById('onoffswitch').checked = backgroundPage.isExtensionEnabled;
+  // Hide or show the rules sections
```

Comment:
Probably makes more sense to set a class on some top-level element within the popup (html, or body?), and have CSS to hide elements when that class is present.